index.html

```diff
@@ -673,17 +673,22 @@ <h2>Expiration</h2>
 
       <section>
         <h2>Status</h2>
-        <p>Credential status information, such as suspension or revocation,
-        may be provided by adding the following <a>property</a>:
+        <p>Information about the current status of a <a>verifiable credential</a>, 
+        such as suspension or revocation,
+        may be provided by adding the credentialStatus <a>property</a>. This property
+       comprises the type of credential status information that is being provided
+ (sometimes referred to as the credential status scheme), plus the id of the status
+type instance. The precise contents of the credential status information is
+           determined by the specific credentialStatus type definition, and will vary
```

Comment:
nit: credentialStatus needs to be wrapped in &lt;code> elements.

Reply:
done